### PR TITLE
- Create Confirmation Dialog UI

### PR DESCRIPTION
**Pull request by danacealfonso**
# 📲 What #

* Confirmation Dialog UI

# ✅ Acceptance criteria #

- [x ] Tested it locally?

# ⏰ TODO #

* Create confirmation dialog UI

# 🔗 References #

Screen Shot 2021-05-25 at 10 14 21 AM

### CODE REVIEW

**Review by MarvinBlanca — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -15,6 +15,8 @@ android {
         targetSdkVersion 30
         versionCode 1
         versionName "1.0"
+        renderscriptTargetApi 22
```
Comment (kalvinbdr) [inline app/build.gradle]: @danacealfonso 
Please refrain from using third-party libraries without consultation. Check with all of the members first.

Comment (danacealfonso) [reply app/build.gradle]: Apologies for implementing third-party libraries. Reverted to f1cc47b9d4345d56493409bb9906022ec10412a7

**Review by rexendy — APPROVED** [review]

**Review by jupilot11 — APPROVED** [review]

File: app/src/main/java/com/kwartracker/android/transactions/ui/details/DetailsTransactionFragment.kt [hunk 2]
```diff
@@ -4,14 +4,19 @@ import android.os.Bundle
...
+import androidx.navigation.fragment.findNavController
 import com.kwartracker.android.R
 import com.kwartracker.android.databinding.FragmentDetailsTransactionBinding
 import com.kwartracker.android.transactions.ui.main.TransactionFragment
+import com.kwartracker.android.transactions.ui.main.TransactionFragmentDirections
+import com.kwartracker.android.widgets.ConfirmationDialog
 
 class DetailsTransactionFragment : Fragment() {
     lateinit var binding: FragmentDetailsTransactionBinding
+    private lateinit var confirmationDialog: ConfirmationDialog
```
Comment (kalvinbdr) [inline app/src/main/java/com/kwartracker/android/transactions/ui/details/DetailsTransactionFragment.kt]: @danacealfonso 
This property is initialized without being used.

File: app/src/main/res/layout/dialog_confirmation.xml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <LinearLayout
```
Comment (kalvinbdr) [inline app/src/main/res/layout/dialog_confirmation.xml]: @danacealfonso 
Can you convert it to ConstraintLayout?

File: .idea/misc.xml [hunk 4]
```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
```
Comment (kalvinbdr) [inline .idea/misc.xml]: @danacealfonso 
This file is not supposed to be pushed.

Comment (danacealfonso) [reply .idea/misc.xml]: fixed in 776fc8db9c272df767d20c16e6f8802a0c8f6128
reverted to its original file.
the only problem on this is the space.

File: app/src/main/res/layout/dialog_confirmation.xml (hunk 3 above)
Comment (danacealfonso) [reply app/src/main/res/layout/dialog_confirmation.xml]: Change layouts in 566294d8722aefaa229b9c9c8f481a16fe5f72cf

File: app/src/main/java/com/kwartracker/android/transactions/ui/details/DetailsTransactionFragment.kt (hunk 2 above)
Comment (danacealfonso) [reply app/src/main/java/com/kwartracker/android/transactions/ui/details/DetailsTransactionFragment.kt]: Removed in c35538db83728f800ede18672975ef2b1b4d22c1

File: .idea/misc.xml (hunk 4 above)
Comment (danacealfonso) [reply .idea/misc.xml]: removed in d16615db003888a0ca5f55ab7d485851fcf20f9b

File: app/src/main/res/layout/dialog_confirmation.xml [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    >
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/scrim"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingStart="30sp"
+        android:paddingEnd="30sp"
+        android:paddingBottom="50sp"
```
Comment (kalvinbdr) [inline app/src/main/res/layout/dialog_confirmation.xml]: @danacealfonso 
Use correct unit(dp) for padding dimensions.

Comment (danacealfonso) [reply app/src/main/res/layout/dialog_confirmation.xml]: Change unit in a6bca1a675a3acbf93e208e450a0966b79bea1c5

**Review by kalvinbdr — APPROVED** [review]

**Pull request merged**